src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -476,6 +477,14 @@
   &:has(.#{$menu-toggle}__controls) {
     padding-inline-start: 0;
   }
+
+  &.pf-m-text {
+    --#{$menu-toggle}--m-split-button--last-child--PaddingLeft: var(--#{$menu-toggle}__button--m-text--PaddingInlineStart);
```

Comment:
`--#{$menu-toggle}--m-split-button--last-child--PaddingLeft` isn't defined or used anywhere - is there a reason we need it or can we remove it?

Reply:
I think this must have been something in v5 that got changed/removed in v6, so will remove the line.